client/stylesheets/_modal.scss

```diff
@@ -233,14 +233,55 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	History Details Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
 
+/*~~ Button End ~~*/
+
+#profileHistoryFeedback {
+	width: 100% !important;
+    background-color: #f8504b !important;
+    padding: 15px 0 !important;
+    text-align: center !important;
+    border-radius: 4px !important;
+    color: #fff !important;
+	text-decoration: none !important;
+}
 
+#profileHistoryFeedback:hover {
```

Comment:
Avoid using id selectors<br>Selector `profileHistoryFeedback` should be written in lowercase with hyphens